tests/test_basic.py

```diff
@@ -64,3 +64,19 @@ def test_iterator_initialization(self) -> None:
         self.assertTrue(ordering.compare(2, 0))
         self.assertTrue(ordering.compare(2, 1))
         self.assertTrue(ordering.compare(0, 1))
+
+    def test_reverse_iter(self) -> None:
+        items: list
+        for items in [], [0], [0, 1], [2, 0, 1]:  # type: ignore # https://github.com/python/mypy/issues/2255
```

Comment:
Extract your test lists to a variable, then you can do `test_lists: List[List[int]] = [...]`